src/main/java/seedu/address/model/profile/Profile.java

```diff
@@ -97,7 +97,8 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
+        builder.append("\nName: ")
```

Comment:
Perhaps it's better to put the new line within the `MESSAGE_SUCCESS` of each command instead in this Profile builder.